doc/database-model.md

```diff
@@ -0,0 +1,61 @@
+
+# Database Entity Relationship Diagram
+
+```mermaid
+erDiagram
+    USER {
```

Comment:
From business a user avatar is a requirement

Reply:
good :+1: